test/projectile-test.el

```diff
@@ -615,25 +618,34 @@ Just delegates OPERATION and ARGS for all operations except for`shell-command`'.
   (it "identifies the root directory of a project by bottom-up search"
     (projectile-test-with-sandbox
      (projectile-test-with-files
-      ("projectA/.svn/"
+      ("projectA/.git/"
+       "projectA/.projectile"
+       "projectA/.svn/"
        "projectA/src/.svn/"
-       "projectA/src/html/.svn/"
-       "projectA/.git/"
-       "projectA/src/html/"
-       "projectA/src/framework/lib/"
        "projectA/src/framework/framework.conf"
-       "projectA/src/html/index.html"
-       "projectA/.projectile")
+       "projectA/src/framework/lib/"
+       "projectA/src/html/"
+       "projectA/src/html/.svn/"
+       "projectA/src/html/index.html")
       (expect (projectile-root-bottom-up "projectA/src/framework/lib" '(".git" ".svn"))
+              :to-equal
+              (expand-file-name "projectA/src/"))
```

Comment:
If you copy-paste the same test to `projectile-root-top-down`, then it will pass as well. This is wrong, as both functions should traverse file system in the opposite direction.

The expected result here should be `"projectA/"`.